src/components/__tests__/Subthoughts.ts

```diff
@@ -395,6 +395,111 @@ describe('hidden thoughts', () => {
     expect(nestedMetaThought2()).toHaveLength(1)
     expect(nestedMetaThought2Child()).toHaveLength(1)
   })
+
+  it('do not hide meta attribute thought when it is the descendant of the meta cursor.', () => {
+
+    // import thoughts
+    store.dispatch(importText({
+      path: HOME_PATH,
+      preventSetCursor: true,
+      text: `
+        - a
+          - =meta1
+            - =meta2`,
+    }))
+
+    // update DOM
+    wrapper.update()
+
+    /* eslint-disable jsdoc/require-jsdoc */
+    const metaThought2 = () => wrapper
+      .find(Subthoughts)
+      .filterWhere(wherePath(['=meta2']))
+
+    // meta2 should should not be visible when it doesn't lie in cursor path or is not descendant of meta cursor
+    expect(metaThought2()).toHaveLength(0)
+
+    store.dispatch(setCursorFirstMatchActionCreator(['a', '=meta1']))
+
+    wrapper.update()
+
+    // meta2 should should be visible when it is descendant of the meta cursor
+    expect(metaThought2()).toHaveLength(0)
```

Comment:
This is testing if meta2 is visible, and yet it asserts `.toHaveLength(0)`. Don't we want it to have length 1?